### PR TITLE
Glowsticks

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
add: Added glowsticks. Found in maintenance, emergency toolboxes and Party Crates.
/:cl:

image

Glowsticks come in green, red, blue, yellow, orange and pink, and use the existing LIGHTING_COLOR for both their produced light and object color.
image
Uncolored glowsticks, note that the last sprite has been renamed to "glowstick-glow" to avoid dupe names.

Code and sprites ~~stolen~~ graciously acquired from Bay.

### CODE REVIEW

**Comment by Cheridan** [comment]
those are extremely bright glowsticks

**Comment by XDTM** [comment]
Now i want colored eswords

Would changing color constantly for hacked ones be too expensive?

File: code/game/objects/items/devices/flashlight.dm [hunk 1]
```diff
@@ -297,3 +299,97 @@
...
 		user << "<span class='warning'>\The [src] needs time to recharge!</span>"
 	return
+
+// Glowsticks, in the uncomfortable range of similar to flares,
+// but not similar enough to make it worth a refactor
+/obj/item/device/flashlight/glowstick
+	name = "green glowstick"
+	desc = "A military-grade glowstick."
+	w_class = WEIGHT_CLASS_SMALL
+	brightness_on = 4
```
Comment (ChangelingRain) [inline code/game/objects/items/devices/flashlight.dm]: Make it 2; with the lighting refactor this is just the range value now

**Comment by feemjmeem** [comment]
but can you rave with them

asking for a friend

**Comment by Supermichael777** [comment]
Can you drink them?

asking for a friend

**Comment by Niknakflak** [comment]
can I pour them on my naked body?

asking for a friend

**Pull request merged**